<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="preferencesTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>320</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>Font</string>
    </attribute>
    <widget class="QLabel" name="fontLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Font Family:</string>
     </property>
    </widget>
    <widget class="QLabel" name="weightLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Font Weight:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="fontList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="fontText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="weightList">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>91</width>
       <height>181</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <item>
      <property name="text">
       <string>Thin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extra Light</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Light</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Demi Bold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extra Bold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="styleLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Font Style:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="styleList">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>91</width>
       <height>181</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Oblique</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="spacingLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Font Spacing (%):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="spacingText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="fontRestoreButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>230</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore Default</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Note: Most fonts do not support every weight and style option</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="coloursTab">
    <attribute name="title">
     <string>Colours</string>
    </attribute>
    <widget class="QTabWidget" name="colourTab">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>301</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="text">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Text Colour</string>
       </property>
      </widget>
      <widget class="QLabel" name="redLabel1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
      <widget class="QLabel" name="greenLabel1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
      <widget class="QLabel" name="blueLabel1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
      <widget class="QSlider" name="redSlider1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="greenSlider1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="blueSlider1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="red1Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="green1Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="blue1Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="restore1Button">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore Default</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="background">
      <attribute name="title">
       <string>Background</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Background Colour</string>
       </property>
      </widget>
      <widget class="QLabel" name="redLabel2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
      <widget class="QLabel" name="greenLabel2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
      <widget class="QLabel" name="blueLabel2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="restore2Button">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore Default</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="red2Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="green2Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="blue2Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="redSlider2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="greenSlider2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="blueSlider2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="highlight1">
      <attribute name="title">
       <string>Highlight 1</string>
      </attribute>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Primary Highlight Colour</string>
       </property>
      </widget>
      <widget class="QLabel" name="redLabel3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
      <widget class="QLabel" name="greenLabel3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
      <widget class="QLabel" name="blueLabel3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
      <widget class="QSlider" name="redSlider3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="restore3Button">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore Default</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="blue3Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="blueSlider3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="greenSlider3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="red3Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="green3Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="highlight2">
      <attribute name="title">
       <string>Highlight 2</string>
      </attribute>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Secondary Highlight Colour</string>
       </property>
      </widget>
      <widget class="QSlider" name="blueSlider4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="restore4Button">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore Default</string>
       </property>
      </widget>
      <widget class="QLabel" name="greenLabel4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="green4Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="red4Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="redSlider4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="blue4Text">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="blueLabel4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
      <widget class="QSlider" name="greenSlider4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="redLabel4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="directoryTab">
    <attribute name="title">
     <string>Directory</string>
    </attribute>
    <widget class="QLabel" name="directoryLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Set the Default Open/Save File Directory</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="currentRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Follow current document</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="chooseRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="chooseText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>301</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
